.github/workflows/cygwin.yml

```diff
@@ -49,10 +49,12 @@ jobs:
   test-cygwin:
     runs-on: windows-latest
     name: Python 3.${{ matrix.python-minor-version }} on Cygwin
+    # Enable these when Cygwin has Python 3.12.
     if: >-
       github.event_name == 'workflow_dispatch' ||
-      github.event_name == 'schedule' ||
+      (false && github.event_name == 'schedule') ||
```

Comment:
Did you mean to keep the `false` here?

Reply:
answer: yes, cygwin is skipping 3.10 and 3.11, but is not ready for 3.12, so will be re-enabled later